pkg/machine/compression/testfiles/sample.uncompressed

```diff
@@ -0,0 +1 @@
+uncompressed
```

Comment:
The canonical Go directory for files like this is `testdata`; that’s hard to find but ultimately documented in e.g. https://go-review.googlesource.com/c/go/+/23353 .

Reply:
done